recipe/activate-cross-python.sh

```diff
@@ -28,6 +28,11 @@ if [[ "${CONDA_BUILD:-0}" == "1" && "${CONDA_BUILD_STATE}" != "TEST" ]]; then
     # For recipes using {{ PYTHON }}
     cp $BUILD_PREFIX/venv/cross/bin/python $PREFIX/bin/python
 
+    # don't set LIBRARY_PATH
+    # See https://github.com/conda-forge/matplotlib-feedstock/pull/309#issuecomment-972213735
+    sed -i 's/extra_envs = .*/extra_envs = []/g' $PREFIX/bin/python        || true
+    sed -i 's/extra_envs = .*/extra_envs = []/d' $PREFIX/bin/python$PY_VER || true
```

Comment:
I'm having trouble grasping the intent of the implementation. Why
1. is it allowed to fail with `|| true`?
2. are there `g`/`d` modifiers?
   1. `g` should have no effect since `.*` catches the rest of the line.
   2. `d` would only makes sense without the `s`, i.e., as a command, not option.

I noticed 2.ii. in a build log:
```
Setting up cross-python
sed: -e expression #1, char 35: unknown option to `s'
Finished setting up cross-python
```